code/Model/SiteTree.php

```diff
@@ -270,6 +270,11 @@ class SiteTree extends DataObject implements PermissionProvider, i18nEntityProvi
         'VirtualPages',
     ];
 
+    private static $cascade_duplicates = [
```

Comment:
Seems like this should actually go directly on `InheritedPermissionsExtension` rather than SiteTree?